examples/generate/src/main.rs

```diff
@@ -0,0 +1,32 @@
+#[allow(warnings, unused)]
```

Comment:
What's the warning? This is not unused is it?

Reply:
This is related to #50. One example of this is all contract methods get generated, but in this example they are not all used. There are a couple other warnings as well. This `allow` attribute should move into the `ethcontract-generate`. I did see your comment there and will narrow down the ignored warnings to use as few of them as possible.

Also note that #50 removes this `allow` attribute completely in favour of more selective `allow` attributes added to the generated code.